### ISSUE TITLE
npm install doesn't work

### ISSUE THREAD

**Issue by TTV-metalrobotz** [rivor/fortnite-scrim-bot#22]
when I do npm install I get this
https://gyazo.com/caab28e64e08eeb128e27b83b21b99a3
https://gyazo.com/b57d77f86dd6c16f24571b97c28c1950

**Comment by AuguxX** [comment]
No, it's a ffmpeg-binaries issue but ignore that.

**Comment by DieselGaming** [comment]
You don't have `Python 2.7+`

**Comment by ic3dr460n** [comment]
I'm having this same issue during npm i.

I'm using latest nodejs and have python 2.7+ installed.

**Comment by DieselGaming** [comment]
Download wbt (Windows build tools)
Personally, I removed the `ffmpeg-binaires` and replaced it with just `ffmpeg`, and that worked fine!
But what I ended up doing (I know how to code in discord.js and I have been doing so for a while now so I usually know what I'm doing) is making the voice count and turning it into a text count, much more easier and the bigger servers happen to use text-counts now anyways.

**Comment by ic3dr460n** [comment]
DieselGaming, do you want to share what you have? I had an odd bug where you sent the bot a PM it would crash. I did fix that. Rivor doesn't update anymore sadly.

**Comment by DieselGaming** [comment]
Yeah it’s because of the constant spam of people asking for help. I feel sorry for him now and trust me it does get frustrating with so many `newbies` are asking stupid questions. I was one of them but then I learnt the language and completely changed this bot. I might upload a better version of this bot on my GitHub and keep updating it and providing support on it. I will only do so if Rivor let’s me of course. Add me on Discord `DieselGaming67#1689` For small snippets though like that text-count.

**Comment by ic3dr460n** [comment]
I'm not that great and probably could fall into that category of asking stupid questions myself but I want to learn its just understanding. I've never done anything with nodejs. I have a few people that help me with it when they can though.

**Comment by ic3dr460n** [comment]
You have to add me apparently. ic3dr460n#1337

It won't let me add you. You have a powerful samurai defending you. ;)
